src/esp/scene/SceneGraph.h

```diff
@@ -27,11 +27,11 @@ class SceneGraph {
   SceneNode& getRootNode() { return rootNode_; }
   const SceneNode& getRootNode() const { return rootNode_; }
 
-  gfx::DrawableGroup& getDrawables(const char* groupName = "") {
-    return drawableGroups_.at(std::string{groupName});
+  gfx::DrawableGroup& getDrawables(std::string groupName = "") {
+    return drawableGroups_.at(groupName);
   }
-  const gfx::DrawableGroup& getDrawables(const char* groupName = "") const {
-    return drawableGroups_.at(std::string{groupName});
+  const gfx::DrawableGroup& getDrawables(std::string groupName = "") const {
```

Comment:
Actually, shoudln't these be const refs?

Reply:
I do not think so since reference is rvalue.

Reply:
Wait, it is `const string&` so it would work (compile).
But it will not help since it will construct a string anyway (because the input parameter is a string literal.)